packages/vsce/src/errors/CICSExtensionError.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * This program and the accompanying materials are made available under the terms of the
+ * Eclipse Public License v2.0 which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-v20.html
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Copyright Contributors to the Zowe Project.
+ *
+ */
+
+import { ICICSExtensionError } from "./ICICSExtensionError";
+import { imperative } from "@zowe/zowe-explorer-api";
+import { CicsCmciRestError } from "@zowe/cics-for-zowe-sdk";
+import constants from "../constants/CICS.defaults";
+
+export class CICSExtensionError extends Error {
+  cicsExtensionError: ICICSExtensionError;
+  constructor(error?: any) {
+    super();
+    this.cicsExtensionError = error;
+    this.parseError(this.cicsExtensionError.baseError);
+  }
+
+  parseError(error: any) {
```

Comment:
Maybe don't need to pass this as the method is part of the same class?? Can just use `this.cicsExtensionError.baseError` inside parseError()????
```suggestion
    this.parseError();
  }

  parseError() {
```